Source/JavaScriptCore/bytecode/StructureStubInfo.h

```diff
@@ -431,7 +462,7 @@ class StructureStubInfo {
     // Note that it's always safe to clear this. If we clear it prematurely, then if we see the same
     // structure again during this buffering countdown, we will create an AccessCase object for it.
     // That's not so bad - we'll get rid of the redundant ones once we regenerate.
-    HashSet<BufferedStructure, BufferedStructure::Hash, BufferedStructure::KeyTraits> m_bufferedStructures WTF_GUARDED_BY_LOCK(m_bufferedStructuresLock);
```

Comment:
Can you delete `BufferedStructure`. I don't think it's used anymore?

Reply:
Done.